redhat-developer/app-services-cli
# refactor(kafka-admin): configure client differently when bootstrap URL is localhost

This PR makes it possible to use the topic (and consumer group) commands with the rhoas CLI.

This enables work to begin on implementing the [consumer group commands](https://github.com/bf2fc6cc711aee1a0c2a/cli/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Adata-plane).

### Verification Steps

1. Run `make mock-api/start` to start the mock API.
2. Run `rhoas login --api-gateway=http://localhost:8000` to use the mock API.
2. Run `rhoas kafka use somekafka` to use a Kafka instance (this will always pick the same one, it is not dynamic).
3. Run `rhoas kafka topic list` to list topics. It will show the mock topics.
4. Check if in the left menu the feature X is not so long present.

### Checklist
